### PR TITLE
Add OpenGraph and meta descriptions to pages

### PR DESCRIPTION
**Pull request by davidfischer**
Fixes https://github.com/readthedocs/website/issues/271.

----
📚 Documentation preview 📚: https://readthedocs-about--282.org.readthedocs.build/

### CODE REVIEW

File: content/pages/enterprise.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
   <meta name="slug" content="pricing/enterprise" />
   <meta name="status" content="hidden" />
   <meta name="description"
-    content="Need more than our self-serve plans? Our Enterprise plans gives you more options." />
+    content="Need more than our self-serve plans? Our Enterprise plans gives you more options so we can best support your documentation team." />
```
Comment (agjohnson) [inline content/pages/enterprise.html:11]:
```suggestion
    content="Need more than our self-serve plans? Our enterprise plans give you more options so we can best support your documentation team." />
```

File: content/pages/features.html [hunk 2]
```diff
@@ -1,6 +1,7 @@
 <html>
   <head>
     <meta name="title" content="Product Features" />
+    <meta name="description" content="Read the Docs includes docs as code features such as building docs, hosting them, search, previewing proposed PR changes, and authentication for private docs." />
```
Comment (agjohnson) [inline content/pages/features.html:4]: I usually point out `docs` -> `documentation` to disambiguate between legal/word/technical "docs".

```suggestion
    <meta name="description" content="Read the Docs includes docs as code features such as building and hosting documentation, search, previewing changes with pull request previews, and authentication for private documentation." />
```

File: content/pages/pricing.html [hunk 3]
```diff
@@ -1,10 +1,10 @@
 <html>
   <head>
     <title>Pricing</title>
+    <meta name="description" content="Read the Docs is free for publicly hosted open source projects and has plans to meet the needs of businesses who need more features like single sign-on, private docs, or more build resources." />
```
Comment (agjohnson) [inline content/pages/pricing.html:4]:
```suggestion
    <meta name="description" content="Read the Docs is free for publicly hosted open source projects and has plans to meet the needs of businesses who need more features like single sign-on, private documentation, or more build resources." />
```

**Review by agjohnson — APPROVED** [review]
These look great, thanks for jumping in here!

**Comment by ericholscher** [comment]
Gonna merge this before it goes out of date and gets conflicts.  🎉

**Pull request merged**